executor/common_linux.h

```diff
@@ -251,8 +253,13 @@ static void initialize_netdevices(void)
 	if (!flag_enable_net_dev)
 		return;
 #endif
+#if SYZ_EXECUTOR || SYZ_ENABLE_NETDEV_DEVTYPES || SYZ_ENABLE_NETDEV_DEVNAMES || SYZ_ENABLE_NETDEV_DEVMASTERS
```

Comment:
This #if looks excessive. We always need this i variable if we generate this function, right?